### PR TITLE
BD-2794 Release notes template

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-2794**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_home/templates/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+nav_title: Release Notes
+page_order: 0
+noindex: true
+page_type: reference
+description: "This article contains a template for release notes."
+permalink: /home/Templates/release_notes/
```
Comment (rachel-feinberg) [inline _docs/_home/templates/release_notes.md:7]:
```suggestion
permalink: /home/Templates/release_notes/
hidden: true
```

File: _docs/_home/templates/release_notes.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
```
Comment (bre-fitzgerald) [inline _docs/_home/templates/release_notes.md:7]: @rachel-feinberg is templates intended to be capitalized here? I know that can sometimes impact link pathing

Comment (rachel-feinberg) [inline _docs/_home/templates/release_notes.md:7]:
```suggestion
permalink: /home/templates/release_notes/
```

Comment (rachel-feinberg) [reply _docs/_home/templates/release_notes.md]: @bre-fitzgerald Oh, right. I updated the pathing. The link works now, but the "Templates" nav header is gone. 🤔 How can I bring it back, assuming there will be more templates?

Comment (bre-fitzgerald) [reply _docs/_home/templates/release_notes.md]: Taking a look 👀

**Review by internetisaiah — APPROVED** [review]
LGTM. Thanks for adding this! 🙌🏽

**Pull request merged**